app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,12 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def role_allows_feature?
+    return role_allows?(:feature => self[:child_id]) unless self[:child_id].nil?
+    return role_allows?(:feature => self[:id]) if self[:items].nil? || self[:items].empty?
+    true
```

Comment:
`self[:items].nil? || self[:items].empty?`

this indicates, that the button is a drop-down?

if so, then we should make it more explicit